### PR TITLE
RAC-6045 Fix Redfish Compliance - /redfish/v1/Systems failures

### PR DESCRIPTION
**Pull request by derrickostertag**
Fix SecureBoot for non-Dells

Work in progess - DO NOT MERGE

Paired with @gpaulos & @jimturnquist

### CODE REVIEW

File: data/views/redfish-1.0/redfish.1.0.0.computersystem.1.0.0.json [hunk 1]
```diff
@@ -60,9 +59,6 @@
     "SimpleStorage": {
         "@odata.id": "<%= basepath %>/Systems/<%= identifier %>/SimpleStorage"
     },
-    "Storage": {
```
Comment (amymullins) [inline data/views/redfish-1.0/redfish.1.0.0.computersystem.1.0.0.json]: Curious as to why this is being removed when it was added ~1 month ago in this pr: https://github.com/RackHD/on-http/pull/708/files.  

original tracking story: https://rackhd.atlassian.net/browse/RAC-5361

Comment (jimturnquist) [reply data/views/redfish-1.0/redfish.1.0.0.computersystem.1.0.0.json]: Storage should only exist in redfish computer systems that are version 1.1.1 or greater.  1.0.0 should not have that field.  The validation tests pick this up as an error.  Also, this is a WIP so I'm guessing its not ready for prime time.

Comment (amymullins) [reply data/views/redfish-1.0/redfish.1.0.0.computersystem.1.0.0.json]: gotcha and thanks for clarification!  by chance, is there a story tracking the update to the 1.1.1 version?

Comment (jimturnquist) [reply data/views/redfish-1.0/redfish.1.0.0.computersystem.1.0.0.json]: No story specifically to transitioning ComputerSystem to 1.1.1.  The fields that are added for 1.1.1 are not required fields so there wouldn't be 1.1.1 conformance issues due to lack of support.

**Comment by jimturnquist** [comment]
Depends on: https://github.com/RackHD/RackHD/pull/882

**Comment by jimturnquist** [comment]
Test this please.

**Comment by jimturnquist** [comment]
Test this please.

**Pull request merged**